tieredstore/src/main/java/org/apache/rocketmq/tieredstore/file/FlatMessageFile.java

```diff
@@ -178,16 +178,20 @@ public AppendResult appendConsumeQueue(DispatchRequest request) {
         return consumeQueue.append(buffer, request.getStoreTimestamp());
     }
 
-
-
     @Override
     public void release() {
-
     }
 
     @Override
     public long getMinStoreTimestamp() {
```

Comment:
Add some java doc here, for using min timestamp in metadata instead of read commitLog from tiered stortge.

Reply:
already add to TieredMessageStore#getEarliestMessageTimeAsync